tests/model_inheritance/tests.py

```diff
@@ -458,3 +460,22 @@ def test_unique_together(self):
         msg = 'Grand parent with this First name and Last name already exists.'
         with self.assertRaisesMessage(ValidationError, msg):
             grand_child.validate_unique()
+
+
+@isolate_apps('model_inheritance')
+class ModelWithInitSubclassTests(TestCase):
+    @skipIf(version_info < (3, 6), 'not supported in python < 3.6')
```

Comment:
I've suggested https://github.com/django/django/pull/9232 to avoid redundant `version_info` in Django.